### PR TITLE
Add files for tideways-proxy and tideways-daemon

### PR DESCRIPTION
**Pull request by alcaeus**

### CODE REVIEW

File: tideways-proxy/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+
+extra="${PROXY_OPTIONS} --hostname=${HOSTNAME:-tideways}"
+extra="${extra} --listen=0.0.0.0:${TIDEWAYS_PORT}"
```
Comment (frag-seb) [inline tideways-proxy/entrypoint.sh]: The beautiful spelling would be`extra+=" --listen=0.0.0.0:${TIDEWAYS_PORT}"`

File: tideways-daemon/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+extra="${DAEMON_OPTIONS} --hostname=${HOSTNAME:-tideways}"
+extra="${extra} --env=${TIDEWAYS_ENV:-development}"
```
Comment (frag-seb) [inline tideways-daemon/entrypoint.sh]: The beautiful spelling would be

`extra+=" --env=${TIDEWAYS_ENV:-development}"`

**Review by frag-seb — APPROVED** [review]
LGTM!

File: tideways-proxy/entrypoint.sh (hunk 1 above)
Comment (alcaeus) [reply tideways-proxy/entrypoint.sh]: Ah, didn't know that also works - will change it 👍

**Comment by alcaeus** [comment]
@frag-seb I had to revert the entrypoint changes as they weren't working in the docker images - should've tested it before merging it 🙈

**Pull request merged**